client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -46,8 +48,22 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 		};
 	}
 
+	handleAddBusinessAddress = () => {
+		const { siteId } = this.props;
+
+		this.props.saveJetpackOnboardingSettings( siteId, {
```

Comment:
Could we use `this.state` here directly? We'd need to rename `stateName` to `state` though (which I personally don't mind).

Reply:
Thank you for reviewing @tyxla 👍 
Could you elaborate please on the last bit? Not sure I see the Why behind it 🤔

Reply:
Speculating -- probably because it'd allow us to write things more tersely?

```js
this.props.saveJetpackOnboardingSettings( siteId, { businessAddress: this.state } );
```

Reply:
Bernie guessed it 👍 

Basically, this allows us to remove some boilerplate code that lists variables 😉  Not to mention this would be more sustainable if we decide to edit the business address fields in the future.

Reply:
Great point!